Game/MyProject/Game/Helper/HelperManager.cpp

```diff
@@ -0,0 +1,553 @@
+        vertices.push_back(vertex2);
+        vertices.push_back(vertex3);
+        vertices.push_back(vertex4);
+        
+        vertex1.position=points[4];
+        vertex2.position=points[5];
+        vertex3.position=points[6];
+        vertex4.position=points[7];
+        
+        vertices.push_back(vertex1);
+        vertices.push_back(vertex2);
+        vertices.push_back(vertex3);
+        vertices.push_back(vertex4);
+        
+        vertex1.position=points[8];
+        vertex2.position=points[9];
+        vertex3.position=points[10];
+        vertex4.position=points[11];
+        
+        vertices.push_back(vertex1);
+        vertices.push_back(vertex2);
+        vertices.push_back(vertex3);
+        vertices.push_back(vertex4);
+        
+        
+        indices.push_back(0);
+        indices.push_back(11);
+        indices.push_back(5);
+        
+        indices.push_back(0);
+        indices.push_back(5);
+        indices.push_back(1);
+        
+        indices.push_back(0);
+        indices.push_back(1);
+        indices.push_back(7);
+        
+        indices.push_back(0);
+        indices.push_back(7);
+        indices.push_back(10);
+        
+        indices.push_back(0);
+        indices.push_back(10);
+        indices.push_back(11);
+        
+        
+        indices.push_back(1);
+        indices.push_back(5);
+        indices.push_back(9);
+        
+        indices.push_back(5);
+        indices.push_back(11);
+        indices.push_back(4);
+        
+        indices.push_back(11);
+        indices.push_back(10);
+        indices.push_back(2);
+        
+        indices.push_back(10);
+        indices.push_back(7);
+        indices.push_back(6);
+        
+        indices.push_back(7);
+        indices.push_back(1);
+        indices.push_back(8);
+        
+        
+        indices.push_back(3);
+        indices.push_back(9);
+        indices.push_back(4);
+        
+        indices.push_back(3);
+        indices.push_back(4);
+        indices.push_back(2);
+        
+        indices.push_back(3);
+        indices.push_back(2);
+        indices.push_back(6);
+        
+        indices.push_back(3);
+        indices.push_back(6);
+        indices.push_back(8);
+        
+        indices.push_back(3);
+        indices.push_back(8);
+        indices.push_back(9);
+        
+        
+        indices.push_back(4);
+        indices.push_back(9);
+        indices.push_back(5);
+        
+        indices.push_back(2);
+        indices.push_back(4);
+        indices.push_back(11);
+        
+        indices.push_back(6);
+        indices.push_back(2);
+        indices.push_back(10);
+        
+        indices.push_back(8);
+        indices.push_back(6);
+        indices.push_back(7);
+        
+        indices.push_back(9);
+        indices.push_back(8);
+        indices.push_back(1);
+        
+        indices.push_back(10);
+        indices.push_back(11);
+        indices.push_back(5);
```

Comment:
i'm pretty sure you can find a pattern and put all of this in a loop.
this code smells (https://en.wikipedia.org/wiki/Code_smell)